sound/soc/sof/intel/mtl.c

```diff
@@ -305,6 +306,8 @@ static void mtl_dsp_dump(struct snd_sof_dev *sdev, u32 flags)
 	romdbgsts = snd_sof_dsp_read(sdev, HDA_DSP_BAR, MTL_DSP_REG_HFFLGPXQWY + 0x8 * 3);
 	dev_printk(level, sdev->dev, "ROM feature bit%s enabled\n",
 		   romdbgsts & BIT(24) ? "" : " not");
+
+	sof_ipc4_intel_dump_telemetry_state(sdev, flags);
```

Comment:
For IPC3 we only do the full dump in case of `flags & SOF_DBG_DUMP_REGS`, should we do the same with ACE.IPC4?

Reply:
It is still applicable to IPC4 since the fw status register is still at the same location